doc/source/management/data_at_rest_encryption.rst

```diff
@@ -154,34 +273,146 @@ Turning this option off at runtime makes server to create all subsequent
 temporary file-per-table tablespaces unencrypted, but does not turn off
 encryption of system temporary tablespace.
 
-.. note:: To use this option, the keyring plugin must be loaded, otherwise
-   server will give an error message and refuse to create new temporary tables.
+This feature is considered **BETA** quality.
+
+.. note:: To use this option, keyring plugin must be loaded, otherwise server
+   will give error message and refuse to create new temporary tables.
 
 .. variable:: innodb_encrypt_tables
 
-  :version 5.7.21-21: Implemented
-  :cli: ``--innodb-encrypt-tables``
-  :dyn: Yes
-  :scope: Global
-  :vartype: Text
-  :default: ``OFF``
+   :version 5.7.21-21: Implemented
+   :cli: ``--innodb-encrypt-tables``
+   :dyn: Yes
+   :scope: Global
+   :vartype: Text
+   :default: ``OFF``
 
-This feature is considered **BETA** quality.
+The implementation of the behavior controlled by this variable is considered
+**ALPHA** quality.
+
+This variable was ported from MariaDB and then extended to support key rotation. This
+variable has the following possible values:
+
+.. rubric:: ON
+
+New tables are created encrypted. You can create an unencrypted table by using
+the ``ENCRYPTION=NO`` clause to the ``CREATE TABLE`` or ``ALTER TABLE``
+statement.
+
+.. rubric:: OFF
+
+By default, newly created tables are not encrypted. Add the ``ENCRYPTION=NO``
+clause in the ``CREATE TABLE`` or ``ALTER TABLE`` statement to create an
+encrypted table.
 
-This variable has 3 possible values. ``ON`` makes |InnoDB| tables encrypted by
-default. ``FORCE`` disables creation of unencrypted tables. ``OFF`` restores
-the like-before behavior.
+.. rubric:: FORCE
 
-.. note:: ``innodb_encrypt_tables=ON`` still allows to create unencrypted
-   table with ``ENCRYPTED=NO`` statement, and also allows to create unencrypted
-   general tablespace.
+New tables are created encrypted with the master key. Passing ``ENCRYPTION=NO`` to
+``CREATE TABLE`` or ``ALTER TABLE`` will result in an error and the table will
+not be created or altered.
```

Comment:
However, altering a table without passing the encryption parameter (and keeping it unencrypted) is possible - so even if it is set to force, it is possible to alter unencrypted tables without encrypting them in the process.

A later part of the doc implies this, but maybe we could mention this here too?

Reply:
Text updated